common2/gratia/common2/timeutil.py

```diff
@@ -26,7 +28,7 @@
     from gratia.common.debug import DebugPrint
 except ImportError:
     def DebugPrint(val, msg):
-        print ("DEBUG LEVEL %s: %s" % (val, msg))
+        print(("DEBUG LEVEL %s: %s" % (val, msg)))
```

Comment:
```suggestion
        print("DEBUG LEVEL %s: %s" % (val, msg))
```
Looks like the converter didn't notice that it was already using parens...